container-dev/pom.xml

```diff
@@ -78,6 +78,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
`container-onnxruntime-bundle` should be added to `container-dev` if an application is allowed to use onnxruntime directly without adding container-onnxruntime-bundle as direct dependency in pom.xml.

Reply:
We probably want it to be available by default, because the bundle is always added to the container cluster. See also the comment in `ContainerModelBuilder.addModelEvaluationBundles` above.